packages/babel-plugin-proposal-decorators/test/fixtures/2021-12-exported/member-decorator/output.mjs

```diff
@@ -1,7 +1,8 @@
 var _xDecs, _init_x;
 export class A {
+  static [(_xDecs = dec, "_")];
   static {
-    _xDecs = dec;
+    delete this._;
```

Comment:
New PR idea: We can optimize them, potentially reusing the `usesFnContext` diagnostic info of each decorators sequence.